open_schools_platform/user_management/users/tests/services/test_user_create.py

```diff
@@ -0,0 +1,23 @@
+from django.test import TestCase
+from django.core.exceptions import ValidationError
+
+from open_schools_platform.user_management.users.services import create_user
+from open_schools_platform.user_management.users.models import User
+
+
+class UserCreateTests(TestCase):
+    def test_user_with_capitalized_email_cannot_be_created(self):
+        create_user(
+            phone="+79112112943",
+            name="Ivan",
+            password="qwe",
```

Comment:
Change to more i8n name, phone. And password length not small? Why do not raise error?

Reply:
Add to task list